src/resources/generated/account.rs

```diff
@@ -1853,7 +1853,7 @@ pub struct TransferScheduleParams {
 }
 
 #[derive(Clone, Debug, Deserialize, Serialize)]
-#[serde(tag = "object", rename_all = "snake_case")]
+#[serde(untagged, rename_all = "snake_case")]
 pub enum ExternalAccount {
```

Comment:
In the openapi spec, the object tag is in fact used, see line 2292 in the spec for `bank_account`, and 3073 for `card`

Reply:
Edit, ignore I have misunderstood.